luigi/contrib/s3.py

```diff
@@ -388,17 +388,22 @@ def get(self, s3_path, destination_local_path):
         # download the file
         self.s3.meta.client.download_file(bucket, key, destination_local_path)
 
-    def get_as_string(self, s3_path):
+    def get_as_bytes(self, s3_path):
         """
-        Get the contents of an object stored in S3 as a string.
+        Get the contents of an object stored in S3 as bytes
         """
         (bucket, key) = self._path_to_bucket_and_key(s3_path)
-
-        # get the content
         obj = self.s3.Object(bucket, key)
-        contents = obj.get()['Body'].read().decode('utf-8')
+        contents = obj.get()['Body'].read()
         return contents
 
+    def get_as_string(self, s3_path, encoding='utf-8'):
+        """
+        Get the contents of an object stored in S3 as string.
```

Comment:
sorry to be nit-picky, but could you add the param docstring descriptions too?

Reply:
Done